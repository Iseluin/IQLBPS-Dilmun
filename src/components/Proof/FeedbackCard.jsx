import { quotes } from "./assets/FeedbackCardImages";

const FeedbackCard = ({ content, name, title, img }) => (
  <div className="flex flex-col px-5 py-6 rounded-[20px] max-w-[370px] md:mr-10 sm:mr-5 mr-0 my-5 hover:bg-secondary ease-in-out duration-500"
    data-testid="feedbackcard-div">
    <img
      src={quotes}
      alt="double_quotes"
      className="w-[42.6px] h-[27.6px] object-contain"
    />
    <p className="font-poppins font-normal text-[17px] leading-[32px] text-black my-10"
      data-testid="feedback-card-content">
      {content}
    </p>

    <div className="flex flex-row">
      <img src={img} alt={name} className="w-[48px] h-[48px] rounded-full" />
      <div className="flex flex-col ml-4">
        <h4 className="font-poppins font-semibold text-[19px] leading-[32px] text-black"
          data-testid="feedback-card-name">
          {name}
        </h4>
        <p className="font-poppins font-normal text-[15px] leading-[24px] text-left text-dimWhite"
          data-testid="feedback-card-title">
          {title}
        </p>
      </div>
    </div>
  </div>
);

export default FeedbackCard;