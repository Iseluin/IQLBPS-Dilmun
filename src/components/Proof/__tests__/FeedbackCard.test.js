import { render, screen } from '@testing-library/react';
import FeedbackCard from '../FeedbackCard';

test('should render feedbackCard component', () => {
    render(<FeedbackCard />);
    const FeedbackCardElement= screen.getByTestId('feedbackcard-div');
    expect(FeedbackCardElement).toMatchSnapshot();
});

