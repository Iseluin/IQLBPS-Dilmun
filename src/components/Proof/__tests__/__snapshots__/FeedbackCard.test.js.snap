// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render feedbackCard component 1`] = `
<div
  class="flex flex-col px-5 py-6 rounded-[20px] max-w-[370px] md:mr-10 sm:mr-5 mr-0 my-5 hover:bg-tertiary ease-in-out duration-500"
  data-testid="feedbackcard-div"
>
  <img
    alt="double_quotes"
    class="w-[42.6px] h-[27.6px] object-contain"
    src="quotes.svg"
  />
  <p
    class="font-poppins font-normal text-[17px] leading-[32px] text-primary my-10"
    data-testid="feedback-card-content"
  />
  <div
    class="flex flex-row"
  >
    <img
      class="w-[48px] h-[48px] rounded-full"
    />
    <div
      class="flex flex-col ml-4 hover:text-background text-primary"
    >
      <h4
        class="font-poppins font-semibold text-[19px] leading-[32px] "
        data-testid="feedback-card-name"
      />
      <p
        class="font-poppins font-normal text-[15px] leading-[24px] text-left"
        data-testid="feedback-card-title"
      />
    </div>
  </div>
</div>
`;
